src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -1338,6 +1338,13 @@ public class CaseData extends Et1CaseData {
     @JsonProperty("legalrepDocumentCollection")
     private List<DocumentTypeItem> legalrepDocumentCollection;
 
+    @JsonProperty("adrDocumentCollection")
+    private List<DocumentTypeItem> adrDocumentCollection;
```

Comment:
looking at the description, these seem to be new item rather than the existing documentType. I guess a new item and type will need to be created

Documents, short description and date.

Reply:
I used a complex type that has the three you mentioned. For upload and listing in tabs it works fine as far as each document has its own description and date pretty much like in the document tab. The date is editable and possibly can be set it programatically.  Do you think it is a set of documents(one or more) with one description and a date for one upload action?